packages/larva-tokens/build/sportico.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
This file was copied into build for some reason - that happens occasionally (I'm not sure why). Please delete this and apply the requested changes to the sportico.json that is in src/